dashboard/client/src/pages/layout/mainNavContext.ts

```diff
@@ -94,6 +91,9 @@ export const useMainNavState = (): MainNavContextType => {
   };
 };
 
+/**
```

Comment:
Should this comment just be removed? I see the comment 
```
 * Render this component at the top of your page if your page should belong in the Main Nav hierarchy
 * Users cannot call the hook directly because useEffect hooks are called from child -> parent on first render
```
in MainNavPageInfo